fmriprep/workflows/base.py

```diff
@@ -669,6 +673,7 @@ def init_single_subject_wf(subject_id: str):
                     f'outputnode.sphere_reg_{"msm" if msm_sulc else "fsLR"}',
                     'inputnode.sphere_reg_fsLR',
                 ),
+                ('outputnode.t2w_preproc', 'inputnode.t2w_preproc'),  # Optional
```

Comment:
I think we no longer actually connect this up in deeper workflows. It could make sense to pass for FSL registration, but I don't really want to put more effort into that.